src/components/Content/SearchResult/Amount/Amount.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AmountCSS from './Amount.module.css';
+
+
+const Amount = (props) => (
+    <h1 className={AmountCSS.sr_text}>
```

Comment:
It is not a good idea to use h1 element here. Read about the purpose of the element
http://htmlbook.ru/html/h1

Reply:
Fixed h1->div